config/initializers/sidekiq.rb

```diff
@@ -24,4 +24,8 @@ def stringify_keys(other)
   config.server_middleware do |chain|
     chain.add Sidekiq::HoneycombMiddleware
   end
+
+  config.death_handlers << lambda do |job, _ex|
```

Comment:
what is `_ex` here? do we _have_ to pass it in? if so, we might want to call out what it is.

Reply:
I think adding a comment linking to this https://github.com/mperham/sidekiq/wiki/Error-Handling#death-notification would be helpful

Reply:
`_ex` here is the exception. I added a comment with that link for more clarity